HISTORY.md

```diff
@@ -3,7 +3,10 @@
 
   * Require new HISTORY.md entry for each PR.
     ([#171](https://github.com/mlpack/ensmallen/pull/171)).
-    
+
+  * Bugfix for HISTORY.md check
+    ([#172](https://github.com/mlpack/ensmallen/pull/171)).
```

Comment:
Maybe just add this one to the existing entry: `([#171](https://github.com/mlpack/ensmallen/pull/171),[#172](https://github.com/mlpack/ensmallen/pull/171))` to keep `HISTORY.md` clean.

Reply:
Good point; thanks!